src/resilience/exec_space/openMP/OpenMPResParallel.hpp

```diff
@@ -74,6 +73,27 @@ namespace KokkosResilience{
 
 } // namespace KokkosResilience
 
+/*--------------------------------------------------------------------------*/
+/************************* ERROR INSERTION TEST CODE ************************/
+/*--------------------------------------------------------------------------*/
+
+namespace KokkosResilience{
+
+  inline void inject_error_duplicates() {
+
+    if (KokkosResilience::global_error_settings){
```

Comment:
```suggestion
    if (global_error_settings){
```

The namespace qualification is a bit redundant here